src/humancellatlas/data/metadata/api.py

```diff
@@ -155,6 +159,10 @@ def __init__(self, json: JSON) -> None:
         self.publications = set(ProjectPublication.from_json(publication)
                                 for publication in content.get('publications', []))
         self.contributors = {ProjectContact.from_json(contributor) for contributor in content['contributors']}
+        self.insdc_project_accessions = {acc for acc in content.get('insdc_project_accessions', [])}
```

Comment:
`{x for x in l}` is the same as `set(l)`

Reply:
Ah yes, without a condition or modification of the values there is no need for the set comprehension. I will simplify these statements.